bin/main.c

```diff
@@ -143,6 +143,49 @@ clt_psk_cb(void *m, char **username, uint8_t **key)
   return keydup(o, key);
 }
 
+static int
+cert_cb(void *m,
+        char **cert_uri,
+        char **key_uri,
+        char **pin)
+{
+  const options_t *o = m;
+
+  *cert_uri = strdup(o->crtf);
+  if (!*cert_uri)
+    return -1;
+  *key_uri = strdup(o->crtk);
+  if (!key_uri)
+    return -1;
+  *pin = strdup(o->crtkp);
+  if (!pin)
+    return -1;
```

Comment:
Hrm... I know that the key URI can specify the PIN as one of its arguments. I'm not sure how I feel about having a separate PIN parameter.